tests/samples/rename_git.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/added b/moved
+similarity index 85%
+rename from added
+rename to moved
+index a071991..4dbab21 100644
+--- a/added
++++ b/moved
```

Comment:
Wondering if we could detect a rename based only in the unified diff output instead of using the rename from/to added by git output? I mean, if we check source filename vs target filename, and they are different, isn't that enough (and more general for any tool that outputs unified diff data)?
What do you think?

Reply:
So I'm kind of reviving this 3 years later but I recently found myself needed this feature. At this point, the git syntax has become about more than just git, as other programs like mercurial also support this syntax.

Is there any downside to having this enabled all the time? It seems like it's a backwards compatible feature (you can still correctly read "standard" patches), and FWIW it's even forwards compatible (unidiff ignores the new rename syntax gracefully).

Reply:
i have a feeling this repo needs to be forked to be maintained - or the maintainer can transfer the repo to someone who wants to handle it, the last commit was 2 years ago